### PR TITLE
docs: azure add actions and no mfa prompt

### PR DESCRIPTION
**Pull request by hifabienne**

### CODE REVIEW

File: docs/docs/apis/actions.md [hunk 1]
```diff
@@ -104,3 +104,4 @@ ZITADEL supports only the external authentication flow at the moment.
 
 - [Actions concept](../concepts/features/actions)
 - [Actions guide](../guides/manage/customize/behavior)
+- [Actions Marketplace: Find example actions to use in your ZITADEL instance](https://github.com/zitadel/actions)
```
Comment (mffap) [inline docs/docs/apis/actions.md:107]:
```suggestion
- [Actions Marketplace: Find example actions to use in ZITADEL](https://github.com/zitadel/actions)
```

File: docs/docs/concepts/features/actions.md [hunk 2]
```diff
@@ -35,4 +35,5 @@ Within the JavaScript code, you can read and manipulate the state.
 ## Further reading
 
 - [Assign users a role after they register using an external identity provider](../../guides/manage/customize/behavior)
-- [Actions reference](../../apis/actions)
\ No newline at end of file
+- [Actions reference](../../apis/actions)
+- [Actions Marketplace: Find example actions to use in your ZITADEL instance](https://github.com/zitadel/actions)
```
Comment (mffap) [inline docs/docs/concepts/features/actions.md:39]:
```suggestion
- [Actions Marketplace: Find example actions to use in ZITADEL](https://github.com/zitadel/actions)
```

File: docs/docs/guides/integrate/azuread-oidc.md [hunk 3]
```diff
@@ -79,9 +79,48 @@ Once you created the IdP you need to activate it, to make it usable for your use
 
 ![Active AzureAD](/img/guides/azure_zitadel_active.png)
 
+#### Disable 2-Factor prompt
+
+If a user has no 2-factor configured, ZITADEL does ask on a regularly basis, if the user likes to add a new 2-factor for more security.
+To not get this prompt if you are using the Azure AD you have to disable it.
```
Comment (mffap) [inline docs/docs/guides/integrate/azuread-oidc.md:85]:
```suggestion
If you don't want your users to get this prompt when using Azure, you have to disable this feature.
```

File: docs/docs/guides/integrate/azuread-oidc.md [hunk 4]
```diff
@@ -79,9 +79,48 @@ Once you created the IdP you need to activate it, to make it usable for your use
...
+
+1. Go to the login behaviour settings of your instance or organization (depending on if you like to disable it for all or just this organization)
```
Comment (mffap) [inline docs/docs/guides/integrate/azuread-oidc.md:87]:
```suggestion
1. Go to the login behaviour settings of your instance or organization, depending if you like to disable it for all or just a specific organization respectively
```

File: docs/docs/guides/integrate/azuread-oidc.md [hunk 5]
```diff
@@ -79,9 +79,48 @@ Once you created the IdP you need to activate it, to make it usable for your use
...
+2. Set "Multi-factor init lifetimes" to 0
+
+![img.png](../../../static/img/guides/login_lifetimes.png)
+
+#### Create user with verified email
+
+Azure AD does not send the email verified claim in its token.
```
Comment (mffap) [inline docs/docs/guides/integrate/azuread-oidc.md:94]:
```suggestion
Azure AD does not send the "email verified claim" in its token.
```

File: docs/docs/guides/integrate/azuread-oidc.md [hunk 6]
```diff
@@ -79,9 +79,48 @@ Once you created the IdP you need to activate it, to make it usable for your use
...
+Due to that the user will get an email verification mail to verify his email address.
+
+To create the user with a verified email address you can add an action.
```
Comment (mffap) [inline docs/docs/guides/integrate/azuread-oidc.md:97]:
```suggestion
To create the user with a verified email address you must add an action.
```

**Review by livio-a — CHANGES_REQUESTED** [review]
the action will fail... the name of the action currently has to match one of the function names in the action itself

File: docs/docs/guides/integrate/azuread-oidc.md [hunk 7]
```diff
@@ -79,9 +79,48 @@ Once you created the IdP you need to activate it, to make it usable for your use
...
+
+1. Go to the actions of your organization
+2. Create a new action with the email verify script
```
Comment (mffap) [inline docs/docs/guides/integrate/azuread-oidc.md:100]:
```suggestion
2. Create a new action with the following code to set the email to verified automatically
```

File: docs/docs/guides/integrate/azuread-oidc.md [hunk 8]
```diff
@@ -79,9 +79,48 @@ Once you created the IdP you need to activate it, to make it usable for your use
...
+
+1. Go to the login behaviour settings of your instance or organization
+2. Disable login with username and password
+3. Make sure you have only configured AzureAD as external identity provider
+4. If you did all your settings on the organization level make sure to send the organization scope in your authorization request: [scope](../../apis/openidoauth/scopes#reserved-scopes)
+
 ### Test the setup
 
-To test the setup use a incognito mode and browse to your login page. 
+To test the setup use a incognito mode and browse to your login page.
```
Comment (mffap) [inline docs/docs/guides/integrate/azuread-oidc.md:123]:
```suggestion
To test the setup use incognito mode and browse to your login page.
```

**Review by mffap — CHANGES_REQUESTED** [review]

**Comment by hifabienne** [comment]
> the action will fail... the name of the action currently has to match one of the function names in the action itself

I have changed it

**Comment by mffap** [comment]
@hifabienne is this ready? if so, can you re-request a review, so we can get this docs merged, please?

**Comment by hifabienne** [comment]
> @hifabienne is this ready? if so, can you re-request a review, so we can get this docs merged, please?

Yes from my perspective it is. I thought you are not finished with the review sue to the wip review comment.

**Review by adlerhurst — APPROVED** [review]

**Review by livio-a — APPROVED** [review]

**Pull request merged**